src/components/forms/TextInput/TextInput.tsx

```diff
@@ -50,21 +55,82 @@ export const TextInput = ({
       'usa-input--success': isSuccess,
       'usa-input--small': isSmall,
       'usa-input--medium': isMedium,
+      'usa-masked': mask,
     },
     className
   )
 
-  return (
+  const [inputValue, setInputValue] = useState('')
+  const [maskValue, setMaskValue] = useState(mask)
+  const [iValue, setIValue] = useState<React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>>()
```

Comment:
Does this not work? (And sorry, when I said `undefined` earlier, I think `null` is actually the right choice.

```suggestion
  const [iValue, setIValue] = useState<HTMLElement | null>(null)
```

Reply:
No, compiler tells me, `Argument of type 'Element' is not assignable to parameter of type 'SetStateAction<HTMLElement | null>'.ts(2345)` (same result if i type the state as just `Element`

Reply:
yeah idk why i never tried this route, was adhering too closely to the original JS implementation...